src/AmplifyMapLibreRequest.ts

```diff
@@ -99,6 +99,11 @@ export default class AmplifyMapLibreRequest {
    */
   transformRequest = (url: string, resourceType: string): RequestParameters => {
     if (resourceType === "Style" && !url.includes("://")) {
+      if (this.region == undefined) {
+        throw new Error(
+          "AWS region for map is undefined. Please verify that the region is set in aws-exports file or that you are providing an AWS region parameter to createMapLibreMap"
```

Comment:
```suggestion
          "AWS region for map is undefined. Please verify that the region is set in aws-exports.js or that you are providing an AWS region parameter to createMap"
```

(i think `createMap` is more likely to be called than `AmplifyMapLibreRequest.createMapLibreMap`)